src/OpenTelemetry.Exporter.Geneva/GenevaExporterOptions.cs

```diff
@@ -46,7 +48,7 @@ public class GenevaExporterOptions
     /// <summary>
     /// Gets or sets table name mappings.
     /// </summary>
-    public IReadOnlyDictionary<string, string> TableNameMappings
+    public IReadOnlyDictionary<string, string>? TableNameMappings
```

Comment:
Guard check seems to be in conflict with the contract.

Reply:
Ya it is a bit weird but seems to be designed this way. `TableNameMappings == null` means the feature is not enabled. It is `null` by default and you have to set it to something to enable table name mappings. Seems once you set it to something/enable it though you can't `null` it out to turn it back off (due to the `Guard` call). Odd, but that's the way it is currently working and I don't want to change the behavior as part of this 😄